src/Native/build.proj

```diff
@@ -2,6 +2,8 @@
   <PropertyGroup>
     <!-- versioning.targets will import this file, so don't import it again -->
     <DisableImportVersioningProps>true</DisableImportVersioningProps>
+    <!-- Use this flag to ensure native assets take stable version -->
+    <UseStableVersionForNativeAssets>true</UseStableVersionForNativeAssets>
```

Comment:
Does this work? I think the `version.h` file gets generated during this target:

[https://github.com/dotnet/machinelearning/blob/0835c52f64ecdc2225966a6bdadc0a4494b5bec2/build.proj#L33](https://github.com/dotnet/machinelearning/blob/0835c52f64ecdc2225966a6bdadc0a4494b5bec2/build.proj#L33)

Which runs during the build of the root `build.proj` file.

Reply:
The Native dlls under `bin\x64.Debug\Native` get the stable version successfully

Reply:
Ah, OK good. Looks like I was mis-remembering how this was working. The native version files get generate during this proj's build:

https://github.com/dotnet/machinelearning/blob/df2f3fb49d4299925325921ee31c2cdff78ba3b5/src/Native/build.proj#L40

and

https://github.com/dotnet/machinelearning/blob/df2f3fb49d4299925325921ee31c2cdff78ba3b5/src/Native/build.proj#L57

This works out great then.